exercises/reverse-string/canonical-schema.json

```diff
@@ -0,0 +1,54 @@
+{
+  "exercise": "revere-string",
+  "version": "1.0.1",
+  "comments": [
+    "The string argument cases possible matches are passed in as",
+    "individual arguments rather than arrays. Languages can include",
+    "these string argument cases if passing individual arguments is",
+    "idiomatic in that language."
+  ],
+  "cases": [
+    {
+      "description": "no matches",
+      "property": "reverse-string",
+      "subject": "car",
+      "candidates": ["girl", "Japan", "bear", "candy"],
+      "expected": []
+    },
```

Comment:
I don't get this… What is `"candidates"` for? And how can reversing a three letter long string result in an empty array?